Reference/Searching/Examine/pdfindex-multisearcher.md

```diff
@@ -1,75 +1,41 @@
----
-versionFrom: 8.0.0
-versionTo: 8.0.0
+﻿---
+versionFrom: 9.0.0
+versionTo: 10.0.0
 ---
 
 # PDF indexes and multisearchers
 
-:::note
-This document has been verified for Umbraco 8.
-
-If you are using Umbraco 9 or later versions, please refer to the note on the [Examine documentation landing page](index.md) for more details.
-:::
-
 If you want to index PDF files and search for them you will need to use the [UmbracoExamine.Pdf extension package](https://github.com/umbraco/UmbracoExamine.PDF).
 
 ## Installation
 
 Install with Nuget:
-`Install-Package UmbracoCms.UmbracoExamine.PDF`
+`dotnet add package Umbraco.ExaminePDF`
 
 You will then have a new Examine index called "PDFIndex" available.
```

Comment:
```suggestion
This will create a new Examine index called "PDFIndex", which will appear in "Examine Management" under the "Settings" section. Using this index you can start searching the contents of any PDF files uploaded to the media section.

![image](https://user-images.githubusercontent.com/7405322/189886089-d23b45c7-814b-4101-b143-31c5cd9fa655.png)

```

Reply:
Hm the screenshot doesn't seem to play nice in the comment above. You will have to open it explicitly to see it I suppose 😞